sdk/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ set(MCMDP_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include)
 set(MEMIF_DIR ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/libmemif)
 set(MCMDP_SAMPLE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/samples)
 set(PROTO_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../protos)
+set(NLOHMANN_JSON_DIR ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/nlohmann)
```

Comment:
Move upper, after libmemif

Reply:
Fixed in [cb1a6cd](https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/291/commits/cb1a6cdeda31d2f55ca1a317549407cea7df013b)